index.html

```diff
@@ -39,200 +39,230 @@ <h3 class="headline-title2">I'm A Software Developer</h3>
                 what you see and have a project you need coded,
                 don’t hesitate to contact me.
             </p>
-            <ul class="headline-social">
-                <li>
-                    <a class="social" rel="noopener" target="_blank" href="https://www.github.com/mahamadou-nouridine">
-                        <i class="fa-brands fa-github"></i>
-                    </a>
-                </li>
-                <li>
-                    <a class="social" rel="noopener" target="_blank"
-                        href="https://www.linkedin.com/in/mahamadou-nouridine-mamoudou-souley-2b424a1a5">
-                        <i class="fa-brands fa-linkedin-in"></i>
-                    </a>
-                </li>
-                <li>
-                    <a class="social" rel="noopener" target="_blank" href="https://www.facebook.com/nouridine2K">
-                        <i class="fa-brands fa-facebook-f"></i>
-                    </a>
-                </li>
-                <li>
-                    <a class="social" rel="noopener" target="_blank" href="https://twitter.com/Nouridine_Dino">
-                        <i class="fa-brands fa-twitter"></i>
-                    </a>
-                </li>
-                <li>
-                    <a class="social" rel="noopener" target="_blank"
-                        href="https://www.instagram.com/mahamadou_nouridine">
-                        <i class="fa-brands fa-instagram"></i>
-                    </a>
-                </li>
-            </ul>
         </div>
+        <ul class="headline-social">
+            <li>
+                <a class="social" rel="noopener" target="_blank" href="https://www.github.com/mahamadou-nouridine">
+                    <i class="fa-brands fa-github"></i>
+                </a>
+            </li>
+            <li>
+                <a class="social" rel="noopener" target="_blank"
+                    href="https://www.linkedin.com/in/mahamadou-nouridine-mamoudou-souley-2b424a1a5">
+                    <i class="fa-brands fa-linkedin-in"></i>
+                </a>
+            </li>
+            <li>
+                <a class="social" rel="noopener" target="_blank" href="https://www.facebook.com/nouridine2K">
+                    <i class="fa-brands fa-facebook-f"></i>
+                </a>
+            </li>
+            <li>
+                <a class="social" rel="noopener" target="_blank" href="https://twitter.com/Nouridine_Dino">
+                    <i class="fa-brands fa-twitter"></i>
+                </a>
+            </li>
+            <li>
+                <a class="social" rel="noopener" target="_blank" href="https://www.instagram.com/mahamadou_nouridine">
+                    <i class="fa-brands fa-instagram"></i>
+                </a>
+            </li>
+        </ul>
```

Comment:
- [ ] As I mentioned above :point_up: please stick to the Figma design your social media icons are totally different from the Figma design. Check below :point_down: the comparison between your design and Figma design :+1: 

**Your design**
- ![Screenshot from 2023-03-30 00-40-27](https://user-images.githubusercontent.com/45226848/228650603-c3a257e6-8f55-4d8b-a217-e7c43b2b1669.png)

- ![Screenshot from 2023-03-30 00-40-38](https://user-images.githubusercontent.com/45226848/228651154-8635425a-6313-4f25-b96f-5620110a2e53.png)

**Figma design** 
- ![Screenshot from 2023-03-30 00-41-06](https://user-images.githubusercontent.com/45226848/228650630-67130d2f-6894-4bfe-8a72-13c490861e91.png)
- ![Screenshot from 2023-03-30 00-40-49](https://user-images.githubusercontent.com/45226848/228651261-e558ed12-0e57-41a2-9347-453cac9069c7.png)

- [ ] So kindly use the same social media icons as mentioned in Figma design :slightly_smiling_face: